### DISCUSSION TITLE
Unable to farm because the external HDD goes to sleep

### DISCUSSION THREAD

**Discussion by leoliyuanduo** [Chia-Network/chia-blockchain#2580] [Q&A]
My laptop uses USB connecting to my 4TB HDD. This external hard drive with plots tends to go to sleep after idling, which means it is no longer farming. Is there any way I can keep external hard drive awake 24/7? I am using Ubuntu and Windows as OS.

**Comment by dextok** [comment]
```shell
*/10 * * * * /usr/bin/ls /mnt/plot*/*.plot > /tmp/plots
```

正常耕种持续读盘，理论上不会休眠才是，不过我的外接硬盘确实每天晚上都会休眠，怀疑是路由器晚上自动重启，节点重新同步期间硬盘没有 IO 自动休眠了，今天刚加了这个定时任务，回头看效果。

**Reply by leoliyuanduo** [reply]
谢谢，看来你是用ubuntu的，你在 etc/fstab 上加过吗？我farming用的是windows，这两个链接的操作用了之后，我暂时还没有发现HDD休眠，还需要时间观察。
https://technicallyeasy.net/fix-prevent-hard-drives-from-going-sleep-windows-10/
https://techcult.com/how-to-prevent-hard-disk-from-going-to-sleep-in-windows-10/

**Reply by musajoemo** [reply]
Use Windows insomnia.

**Comment by OversizedWizard** [comment]
Goto advanced power options. Usb setting, set to never sleep
